src/main/java/com/example/petbuddybackend/entity/user/Caretaker.java

```diff
@@ -15,21 +17,28 @@
 public class Caretaker {
 
     @Id
```

Comment:
```suggestion
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Wydaje mi sie ze mozna generowac automatycznie, chyba ze z jakiegos powodu potrzeba tak?

Reply:
Zastąpiłem to:

`    @Id
    private String email;
`